### ISSUE TITLE
Request for a method to connect to the tronlink wallet

### ISSUE THREAD

**Issue by frankudoags** [Eryk-Luiz/use-tronlink#3]
Love your work, thank you for this.
Please does the use-tronlink package expose a method to connect to the tronlink wallet?
If no, please can you make one? Thank you.

**Comment by frankudoags** [comment]
Nvm, just went through the code and i understand now, thanks.

**Comment by eryk-vieira** [comment]
@frankudoags For now, just using the hook will request to connect Tronlink.
Even though your suggestion is good, having a method to connect, Will allow connecting only if the user clicks on a Button for example...

Creating a parameter for the hook that you can set if you want the package connects automatically or not may solve it.

**Comment by frankudoags** [comment]
@eryk-vieira  Yes, I saw that the hook connects automatically on load, and I don't really think that's good UX but I don't know how else to do it. I'm also looking for other hooks to prepare a contract and run transactions similar to wagmi hook.I'm new to the tron network.
